Direct anchor to limits in domain model docs, document definition (#2968)
SoftwareAG/c8y-docs: content/glossary/d.md

### Digital twin {#digital-twin}

A digital twin is a digital representation of physical or virtual assets. Devices, sensors and tools are examples of physical assets, virtual assets can for example be processes or rules. In {{< product-c8y-iot >}} they are represented as [managed objects](/glossary/m/#managed-object).

### Document {#document}

> {{< product-c8y-iot >}} stores all operational data --- such as managed objects, alarms and measurements --- as documents. A document is a versatile JSON structure with a unique identifier, allowing it to be referenced in URLs of web applications or REST APIs. This provides a high degree of flexibility for modeling your data on top of the {{< product-c8y-iot >}} [domain model](/concepts/domain-model/) and the {{< product-c8y-iot >}} [service quotas](/service-terms/quotas/).

Reviewer: ```suggestion
{{< product-c8y-iot >}} stores all operational data --- such as managed objects, alarms, events and measurements --- as documents. A document is a versatile JSON structure that provides a high degree of flexibility for modeling your data on top of the {{< product-c8y-iot >}} [domain model](/concepts/domain-model/) and the {{< product-c8y-iot >}} [service quotas](/service-terms/quotas/).
```

I suggest to not mention the unique identifier as a measurement document does currently not have a unique identifier afaik.

Author: We do actually ... e.g. https://cumulocity.com/api/core/#operation/getMeasurementResource Technically, that's correct. Whether you want it in the glossary or not, I leave that up to you ;-) Feel free to commit and merge or approve. Cheers!

Reviewer 2: @eickler Sorry, I have overwritten this comment with another change. @neuerburg What do you suggest, leave it in the glossary or remove it from here?

Reviewer: Then I take everything back and you can ignore this comment!

> {{< product-c8y-iot >}} stores all operational data --- such as managed objects, alarms and measurements --- as documents. A document is a versatile JSON structure with a unique identifier, allowing it to be referenced in URLs of web applications or REST APIs. This provides a high degree of flexibility for modeling your data on top of the {{< product-c8y-iot >}} [domain model](/concepts/domain-model/) and the {{< product-c8y-iot >}} [service quotas](/service-terms/quotas/).

Reviewer 2: ```suggestion
{{< product-c8y-iot >}} stores all operational data - such as managed objects, alarms and measurements - as documents. A document is a versatile JSON structure with a unique identifier, allowing it to be referenced in URLs of web applications or REST APIs. This provides a high degree of flexibility for modeling your data on top of the {{< product-c8y-iot >}} [domain model](/concepts/domain-model/) and the {{< product-c8y-iot >}} [service quotas](/service-terms/quotas/).
```
